app/settings/manifest.js

```diff
@@ -143,6 +143,77 @@ vote at ${issue(125,true)}.
             "type": "checkbox",
             "label": future_i18n('Sort open windows to the top, scroll to the top of the list'),
         },
+        {
+            "tab": future_i18n("Behaviour"),
+            "group": future_i18n("When I..."),
+            "name": S.FIXED_WINDOW_SIZE,
+            "type": "checkbox",
+            "label": future_i18n('Open a window, always open it with a preferred fixed size'),
+        },
+        {   // some extra descriptive text for S.FIXED_WINDOW_SIZE
+            "tab": future_i18n("Behaviour"),
+            "group": future_i18n("When I..."),
+            "type": "description",
+            "text": future_i18n(`If you change this value please refresh Settings
+                    page (this page) to reveal preferred window size parameter`)
+        },
+    ];
+
+    if(S.getBool(S.FIXED_WINDOW_SIZE)) setting_definitions.push(
```

Comment:
Style: please add `{ }` around the `settings_definitions.push` for consistency with in the codebase.